src/Bicep.Core/TypeSystem/Radius/V3/KnownComponents.cs

```diff
@@ -242,5 +276,22 @@ public static ComponentData MakeMongoDB()
                 },
             };
         }
+
+        public static ComponentData MakeVolume()
+        {
+            var configKindType = UnionType.Create(
+                new StringLiteralType("azure.com.fileshare"));
+
+            return new ComponentData()
+            {
+                Type = new ThreePartType(null, "Volume", RadiusResources.CategoryComponent),
```

Comment:
This shows up as 'VolumeComponent' in the bicep file instead of 'Volume' but it seems to be the same for all other component types too (such as KV)

Reply:
Ok. I might have to do some surgery here to support this. I'll add a commit.

Reply:
Can you try passing `""` in place of `RadiusResources.CategoryComponent`. It should just work ;-)